Makefile

```diff
@@ -76,7 +82,9 @@ run-estimator-client:
 		hatch run test -vvv -s ./tests/estimator_model_request_test.py
 
 clean-model-server:
+	@$(CTR_CMD) logs model-server
 	@$(CTR_CMD) stop model-server
+	@$(CTR_CMD) rm model-server || true
```

Comment:
Aren't we already adding --rm to the `run` ? I think we just need to `stop` and the container will be automatically deleted.

Reply:
It should be but it fails to remove sometimes before starting the new test which also need model-server container. It is the reason that I add rm line with || true.